github-status/handler.go

```diff
@@ -153,6 +169,118 @@ func reportStatus(status string, desc string, statusContext string, event *sdk.E
 	return nil
 }
 
+func reportCheck(commitStatus *sdk.CommitStatus, event *sdk.Event) error {
+	ctx := context.Background()
+
+	status := commitStatus.Status
+	url := buildPublicStatusURL(commitStatus.Status, commitStatus.Context, event)
+
+	log.Printf("Check: %s, Context: %s, GitHub AppID: %d, Repo: %s, Owner: %s", status, commitStatus.Context, event.InstallationID, event.Repository, event.Owner)
+
+	client := auth.MakeClient(ctx, token)
+
+	now := github.Timestamp{time.Now()}
+
+	logs, err := getLogs(commitStatus, event)
```

Comment:
I like your idea of having logs for stack context as well - I recently had some issues with function deployment and it would have been nice to see those in GitHub. In this direction, do you think it would be OK to have a failing call, while not having to change this function when the stack context starts writing logs?